### ISSUE TITLE
Linux: Segmentation Fault when starting multiple camera's at the same time.

### ISSUE THREAD

**Issue by wes-b** [microsoft/Azure-Kinect-Sensor-SDK#769]
**Describe the bug**
Related to #676, OpenGL usage on Linux causes a segmentation fault when two threads call into DepthEngine_CreateAndInitialize. 

**To Reproduce**
1. multidevice_ft.exe --gtest_also_run_disabled_tests --gtest_filter="*paral*"

**Expected behavior**
Tests should all pass

**Stack**
```
Thread 9 (Thread 0x7fffeeffd700 (LWP 5116)):
#0  free_exts ()
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/OpenGL/GlFramework/Glad/glad.c:208
#1  find_extensionsGL ()
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/OpenGL/GlFramework/Glad/glad.c:1477
#2  gladLoadGLLoader (load=<optimized out>)
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/OpenGL/GlFramework/Glad/glad.c:1564
#3  0x00007fffee5b70f2 in gladLoadGL ()
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/OpenGL/GlFramework/Glad/glad.c:150
#4  0x00007fffee5bbdff in OpenGLDepthEngine::GlContext::Initialize (this=0x7fffd800db40)
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/OpenGL/GlFramework/GlContext.cpp:334
#5  0x00007fffee5b3464 in OpenGLDepthEngine::GlPipeline::Initialize (this=0x7fffd800dad0)
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/OpenGL/GlFramework/GlPipeline.cpp:37
#6  0x00007fffee5ad474 in OpenGLDepthEngine::GlPipelineDepth::Initialize (this=0x7fffd800dad0, 
    pipelineConfiguration=<optimized out>)
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/OpenGL/GlAlgorithm/GlPipelineDepth.cpp:27
#7  0x00007fffee5aaf8a in GpuBasedDepthEngine::CreateDepthPipeline (pPipeline=@0x7fffd800c938: 0x0, 
    configuration=0x7fffd800eef0)
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/GpuPipelineFactory.cpp:68
#8  0x00007fffee5a9796 in GpuBasedDepthEngine::GpuDepthEngine::InitDepthEngine (this=0x7fffd8005020)
    at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngineLib/Src/GpuDepthEngine.cpp:60
#9  0x00007fffee5a8e0f in DepthEngine_CreateAndInitialize (ppContext=0x5555557dc668, calBlockSizeInBytes=502016, 
    pCalBlock=0x7ffff5b4b010, mode=K4A_DEPTH_ENGINE_MODE_LT_SW_BINNING, 
    inputFormat=K4A_DEPTH_ENGINE_INPUT_TYPE_12BIT_COMPRESSED, pCameraCalibration=0x5555557e1e64, pCallback=0x0, 
    pCallbackContext=0x0) at /home/vsts/work/1/s/extern/analog.ai.depthcamera/src/DepthEngine/DepthEngine.cpp:82
#10 0x00007ffff7ac43ad in depth_engine_start_helper (depth_engine_output_buffer_size=<synthetic pointer>, 
    depth_engine_max_compute_time_ms=<synthetic pointer>, depth_mode=<optimized out>, fps=<optimized out>, 
    dewrapper=0x5555557dc608) at ../../src/dewrapper/dewrapper.c:143
#11 depth_engine_thread (param=0x5555557dc608) at ../../src/dewrapper/dewrapper.c:190
#12 0x00007ffff7b7053e in ThreadWrapper (threadInstanceArg=<optimized out>)
    at ../../extern/azure_c_shared/src/adapters/threadapi_pthreads.c:34
#13 0x00007ffff78966db in start_thread (arg=0x7fffeeffd700) at pthread_create.c:463
#14 0x00007ffff701e88f in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```
**Desktop (please complete the following information):**
 - OS with Version: 18.04
 - SDK Version:  develop

**Comment by rabbitdaxi** [comment]
Fixed depthengine from internal repo, will be available in next release

**Comment by wes-b** [comment]
Opened, so that test can be reenabled

**Comment by wes-b** [comment]
Fixed in Depth Engine Version 2.1.0 which is in 1.3.0
